vickyctl/src/humanize.rs

```diff
@@ -1,9 +1,23 @@
-use crate::error::Error;
-use crate::AppContext;
-use log::debug;
 use std::io::{ErrorKind, Write};
 use std::process::{Command, Stdio};
 
+use log::debug;
+use which::which;
+
+use crate::AppContext;
+use crate::error::Error;
+
+pub fn ensure_jless(needed_for: &str) -> Result<(), Error> {
```

Comment:
Not sure, why this change is in this PR, but it can stay.

Reply:
Well I added some stuff to the cli and refactored this out since it's now used more than once, while before it wasn't.